vocabularies/Org.OData.Temporal.V1.xml

```diff
@@ -67,98 +67,79 @@
       </Annotation>
 
 
-      <!-- Metadata annotations -->
-
-      <Term Name="TemporalSupport" Type="Temporal.TemporalSupportType" AppliesTo="EntitySet NavigationProperty">
+      <Term Name="ApplicationTimeSupport" Type="Temporal.ApplicationTimeSupportType" Nullable="false" AppliesTo="Collection">
+        <Annotation Term="Core.AppliesViaContainer" />
         <Annotation Term="Core.Description">
-          <String>This entity set or navigation property supports  temporal requests</String>
+          <String>This entity set or navigation property supports temporal requests</String>
         </Annotation>
       </Term>
 
-      <ComplexType Name="TemporalSupportType">
-        <Property Name="ApplicationTime" Type="Temporal.TimeDimensionType">
-          <Annotation Term="Core.Description" String="Application time granularity" />
+      <ComplexType Name="ApplicationTimeSupportType">
+        <Property Name="CollectionKind" Type="Temporal.CollectionKindType" Nullable="false">
+          <Annotation Term="Core.Description" String="Kind of temporal collection" />
         </Property>
-        <Property Name="SystemTime" Type="Temporal.GranularityDateTimeOffset">
-          <Annotation Term="Core.Description" String="System time granularity" />
+        <Property Name="PeriodKind" Type="Temporal.PeriodKindType" Nullable="false">
+          <Annotation Term="Core.Description" String="Kind of application time period" />

```

Comment:
```suggestion
          <Annotation Term="Core.Description" String="How the boundaries of an application time period may be defined" />
```

Reply:
It's not only the data type of the boundaries, it can also be additional information such as `ClosedClosedPeriods`